portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewFields.testcase

```diff
@@ -1481,6 +1481,320 @@ definition {
 		}
 	}
 
+	@description = "LPS-187522. Allow administrators to configure client extension filters on dataset views."
```

Comment:
You have to use the LPS from the parent story, in this case LPS-182558. Also, the description makes no sense with the name of the test. In this test you are checking that, when the user clicks on Cancel, the Client Extension filter is not created